src/tests/core.py

```diff
@@ -91,7 +93,7 @@ def test_tcpconnection(self):
             for peer in (state.Peer("127.0.0.1", 8448),):
                 direct = TCPConnection(peer)
                 while asyncore.socket_map:
-                    print("loop, state = %s" % direct.state)
+                    print "loop, state = %s" % direct.state
```

Comment:
print function is used intentionally and it is valid for python 2.7.6 (minimum version supported) Any old style prints in code should be replaced if `print` cannot be avoided there (#1373) in order to prepare for python 3 support.

https://docs.python.org/3/whatsnew/3.0.html#print-is-a-function

Reply:
So should i change it with logger?

Reply:
Better do not touch this broken test. I'm going to find an appropriate environment to run it. The print was in the original script.

Reply:
Okay, wil change "print "loop, state = %s" % direct.state" -> print("loop, state = %s" % direct.state).

Reply:
The print was there for debugging I believe, for tests we can keep it.